docs/compass/08-01-tutorial.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Manage your Applications using Kyma and Compass Console UI
+type: Tutorial
+---
+
+This tutorial present end-to-end use case scenario that shows how to connect an external Application to Compass so that its lambda calls and gets the order services.
+
+## Prerequisites
+
+Use the [HTTP DB Service](https://github.com/kyma-project/examples/tree/master/http-db-service) application to go through this tutorial. Download the following:
+- HTTP DB Service [API definition](https://github.com/kyma-project/examples/blob/master/http-db-service/docs/api/api.yaml)
```

Comment:
I think it is rather more common and straightforward to give the "raw" links where the content can be easily copied -> `https://raw.githubusercontent.com/kyma-project/examples/master/http-db-service/docs/api/api.yaml`. I guess I would also not use the phrase "download" as you actually do not have this option in GitHub.